assets/resources/badusb/Install_qFlipper_macOS.txt

```diff
@@ -0,0 +1,59 @@
+ID 05ac:021e Apple:Keyboard
+REM Keep these 3 lines IF (and only if) it's the first time you are performing a badKB attack against a specific macOS target. 
+REM In fact, it helps Flipper Zero bypass the macOS keyboard setup assistant. Otherwise the attack will not start.
+
+REM Author: 47LeCoste
+REM Version 1.0 (Flipper Ducky)
+REM Target: macOS
+DELAY 300
+GUI SPACE
+DELAY 1000
+STRING Terminal
+DELAY 1000
+ENTER
+DELAY 1000
+STRING curl -L -o /tmp/qFlipper.dmg https://update.flipperzero.one/qFlipper/release/macos-amd64/dmg
```

Comment:
There are so many cases when it's not going to work.

Let's from scratch with couple thing in mind:
- Download time is unpredictable. 6 seconds are not going to help
- Delay based execution is cursed in a same way
- GUI SPACE - are not going to work in some cases (old keyboard layout shortcut)
- TAB not going to behave well

What must be done instead:
- Write standalone shell script that reliably works in all conditions
- on BadUSB side save this script into temporary file and execute it
- Minimize all unnecessary actions and delays

Reply:
hey, i changed everything you asked. now it should be ok. let me know!!

Reply:
i did not made it with a teporary shell script, but it would be ok this way as well, i think..

Reply:
Im thinking if maybe running all with a "nohup" or a "screen -dm" before, could permit to quit the terminal and making the script continue working in background, showing nothing unless qFlipper starting after been moved in /Applications and ready to run .. let me know what do you think also about this :)